service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -34,12 +38,16 @@
     @Autowired
     private DateFormatterService dateFormatterService;
 
+    @Autowired
+    private DirectionHelperService directionHelperService;
+
     private final LocalDate date = LocalDate.now();
 
     @Test
     void shouldReturnHearingDateDynamicListWhenCaseDetailsHasHearingDate() {
         CaseDetails caseDetails = CaseDetails.builder()
-            .data(ImmutableMap.of("hearingDetails", createHearingBookings(date)))
+            .data(ImmutableMap.of(
```

Comment:
does this need to be spaced down?

Reply:
Probably not, you commented earlier about fixing spacing, maybe I went a bit overboard :)